### PR TITLE
Feature/les 58/add currency to vc

### PR DESCRIPTION
**Pull request by mehmetbicak**

### CODE REVIEW

File: les/winnercheque-v1.html [hunk 1]
```diff
@@ -11,13 +11,17 @@ <h4 id="chequeId"><a href="#chequeId">chequeId</a></h4>
...
+The crypto address of the recipient.
 
 <h4 id="monetaryAmount"><a href="#monetaryAmount">monetaryAmount</a></h4>
 
-The amount that should be transferred to the wallet.
+The amount that should be transferred to the crypto address.
+
+<h4 id="monetaryCurrency"><a href="#monetaryCurrency">monetaryCurrency</a></h4>
+
+The currency of the amount that should be transferred to the wallet.
```
Comment (BtencateSphereon) [inline les/winnercheque-v1.html:24]: wallet or crypto address, as this is what we call it everywhere

File: les/winnercheque-v1.jsonld [hunk 2]
```diff
@@ -1,16 +1,16 @@
 {
-  "@context":
-    {
-		"@version": 1.1,
-		"wc": "https://sphereon-opensource.github.io/vc-contexts/les/winnercheque-v1#",
-		"schema": "http://schema.org/",
+  "@context": {
+    "@version": 1.1,
```
Comment (BtencateSphereon) [inline les/winnercheque-v1.jsonld:3]: should the version be bumped for this?

File: les/winnercheque-v1.html (hunk 1 above)
Comment (BtencateSphereon) [reply les/winnercheque-v1.html:24]: The currency code of the transferring amount. ?

File: les/winnercheque-v1.jsonld (hunk 2 above)
Comment (nklomp) [reply les/winnercheque-v1.jsonld:3]: This is a JSON-LD version, not sure what you want to bump there?

File: les/winnercheque-v1.html (hunk 1 above)
Comment (nklomp) [reply les/winnercheque-v1.html:24]: Just address

Comment (mehmetbicak) [reply les/winnercheque-v1.html:24]: CoinBasePro uses currency instead of currency code. So we used currency too. 
Eg. monetaryCurrency: "USDC"

**Review by BtencateSphereon — APPROVED** [review]
LGTM

**Pull request merged**